libs/langchain/langchain/retrievers/multi_vector.py

```diff
@@ -27,12 +28,40 @@ class MultiVectorRetriever(BaseRetriever):
     and their embedding vectors"""
     docstore: BaseStore[str, Document]
     """The storage layer for the parent documents"""
-    id_key: str = "doc_id"
-    search_kwargs: dict = Field(default_factory=dict)
+    id_key: str
+    search_kwargs: dict
     """Keyword arguments to pass to the search function."""
-    search_type: SearchType = SearchType.similarity
+    search_type: SearchType
     """Type of search to perform (similarity / mmr)"""
 
+    def __init__(
```

Comment:
This is a pydantic object. So you use a pre-validator instead of using the __init__. The init might be a bit easier though since it contains `base_store` parameter

Reply:
Ok, will try to switch it